assets/queries/cloudFormation/kms_should_allow_principal/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "kms_should_allow_principal",
+  "queryName": "Kms Should Allow * as Principal",
```

Comment:
Change **query directory name** to: `kms_allows_wildcard_principal`
Change **id** to: `kms_allows_wildcard_principal`
Change **query name** to: `KMS allows wildcard principal`